### PR TITLE
[doc] Fix wrong path in getting started guide

### PR DESCRIPTION
**Pull request by vogelpi**

### CODE REVIEW

File: doc/ug/getting_started_verilator.md [hunk 1]
```diff
@@ -165,8 +165,8 @@ Tracing slows down the simulation by roughly factor of 1000.
 ```console
 $ cd $REPO_TOP
 $ build/lowrisc_systems_top_earlgrey_verilator_0.1/sim-verilator/Vtop_earlgrey_verilator \
-  --meminit=rom,sw/device/sim_boot_rom/rom.elf \
-  --meminit=flash,sw/device/sim_hello_world/sw.elf \
+  --meminit=rom,build-bin/sw/device/sim-verilator/boot_rom/boot_rom.elf \
+  --meminit=flash,build-bin/sw/device/sim-verilator/examples/hello_world/hello_world.elf
```
Comment (imphil) [inline doc/ug/getting_started_verilator.md]: You should use build-out paths here, the build-bin variants don't have a stable directory structure.

Comment (vogelpi) [reply doc/ug/getting_started_verilator.md]: Thanks for pointing that out Philipp!

Comment (mcy) [reply doc/ug/getting_started_verilator.md]: I think you got that backwards, build-out is where Meson dumps stuff, build-bin is where exports happen.

Comment (vogelpi) [reply doc/ug/getting_started_verilator.md]: Ah, okay it is indeed `build-bin` where we should get the binaries from. This is documented here: https://docs.opentitan.org/doc/ug/getting_started_sw/

Comment (imphil) [reply doc/ug/getting_started_verilator.md]: Turns out the naming is indeed confusing.

**Pull request merged**